utils/constants.tsx

```diff
@@ -431,6 +431,7 @@ export const ModalIdentifiers = {
     MARK_ALL_THREADS_AS_READ: 'mark_all_threads_as_read_modal',
     DELINQUENCY_MODAL_DOWNGRADE: 'delinquency_modal_downgrade',
     CLOUD_LIMITS_DOWNGRADE: 'cloud_limits_downgrade',
+    WORKTEMPLATES: 'worktemplate',
```

Comment:
Work template should also be two separate words here as well

Reply:
if this is really important, I'll make a ticket to do that later (but before prod), as we have tickets ongoing based on this branch and changing request/changing those values, I would rather avoid conflict right now and go through a renaming session later (I would also have to rename a bunch on things in the backend)

Reply:
I would +1 on the renaming - given that the other PRs have merged here, should we rename it before merge?

Reply:
I think we should rename it first or else we're going to end up with a mix of code using one and code using the other

Reply:
Well that was a fun refactoring :P 35 files changed, we're all good now!

Reply:
Fun stuff for sure 💯 🙃